static/src/javascripts/bootstraps/atoms/storyquestions.js

```diff
@@ -23,13 +35,15 @@ Promise.all([
     new Promise(comready),
 ]).then(() => {
     init();
-    fastdom
-        .read(
-            () =>
-                document.documentElement &&
-                document.documentElement.getBoundingClientRect().height
-        )
-        .then(height => {
-            send('resize', { height });
+    updateHeight();
+
+    // Brittle but will work
+    [...document.getElementsByClassName('user__question')]
```

Comment:
because the return type is `HTMLCollection`, it is not an array but a weird mutable collection. I should've written

`Reflect.apply(Array.prototype.slice, document.getElementsByClassName(...), [0, 1])`

but arguably that is even uglier.